app/jobs/make_phone_calls_job.rb

```diff
@@ -0,0 +1,28 @@
+class MakePhoneCallsJob < ApplicationJob
+  queue_as :phone_calls
+
+  require 'net/http'
+  require 'open-uri'
+
+  def perform(args)
+    # Find all the phone stores for which there an activecalendar for the current day of the week
+    # For each store, create a job to make the request
+
+    Rails.logger.debug("Going to make a phone call to #{args[:phone]}")
+
+    phone = args[:phone]
+    store = args[:store] || '12345' # random number in case there's no store
```

Comment:
This, as a default behavior, seems dangerous. Don't you want to blow up instead, by using `args.fetch(:store)`?

Reply:
For now, the store_id is not mandatory, so any number will work. But yeah, true.